src/lib/format/tests/TestFlatTreePosition.cpp

```diff
@@ -94,16 +93,16 @@ class ByName
 };
 
 #define ASSERT_HAS_NAME(p, n)                                                                                                      \
-    NL_TEST_ASSERT(inSuite, p.Get() != nullptr);                                                                                   \
-    NL_TEST_ASSERT(inSuite, strcmp(p.Get()->name, n) == 0)
+    ASSERT_NE(p.Get(), nullptr);                                                                                                   \
```

Comment:
nit: remove some trailing whitespace

Reply:
I just realized this may be github displaying things on multiple lines.... this is probably ok as clang-format should make things pretty by default.